bundles/org.openhab.binding.nikohomecontrol/README.md

```diff
@@ -237,19 +237,25 @@ OnOff, IncreaseDecrease and Percent command types are supported.
 Note that sending an ON command will switch the dimmer to the value stored when last turning the dimmer off, or 100% depending on the configuration in the Niko Home Control Controller.
 This can be changed with the Niko Home Control programming software.
 
-For thing type `blind` the supported channel is `rollershutter`. UpDown, StopMove and Percent command types are supported.
+For thing type `blind` the supported channel is `rollershutter`.
+UpDown, StopMove and Percent command types are supported.
 
-For thing type `thermostat` the supported channels are `measured`, `mode`, `setpoint`, `overruletime` and `demand`.
+For thing type `thermostat` the supported channels are `measured`, `heatingmode`, `mode`, `setpoint`, `overruletime`, `heatingdemand` and `demand`.
 `measured` gives the current temperature in QuantityType<Temperature>, allowing for different temperature units.
 This channel is read only.
-`mode` can be set and shows the current thermostat mode.
-Allowed values are 0 (day), 1 (night), 2 (eco), 3 (off), 4 (cool), 5 (prog 1), 6 (prog 2), 7 (prog 3).
-If mode is set, the `setpoint` temperature will return to its standard value from the mode.
-`setpoint` can be set and shows the current thermostat setpoint value in QuantityType<Temperature>.
+`heatingmode` can be set and shows the current thermostat mode.
+Allowed values are Day, Night, Eco, Off, Cool, Prog1, Prog2, Prog3.
+As an alternative to `heatingmode` and for backward compatibility, the advanced channel `mode` is provided.
+This channel has the same meaning, but with numeric values (0=Day, 1=Night, 2=Eco, 3=Off, 4=Cool, 5=Prog1, 6=Prog2, 7=Prog3) instead of string values.
+If `heatingmode` or `mode` is set, the `setpoint` temperature will return to the standard value for the mode as defined in Niko Home Control.
+`setpoint` shows the current thermostat setpoint value in QuantityType<Temperature>.
 When updating `setpoint`, it will overrule the temperature setpoint defined by the thermostat mode for `overruletime` duration.
-`overruletime` is used to set the total duration to apply the setpoint temperature set in the setpoint channel before the thermostat returns to the setting in its mode.
-`demand` is a number indicating of the system is actively heating/cooling.
+`overruletime` is used to set the total duration to apply the setpoint temperature set in the setpoint channel before the thermostat returns to the setting from its mode.
+`heatingdemand` is a string indicating if the system is actively heating/cooling.
+This channel will have value Heating, Cooling or None.
+As an alternative to `heatingdemand` and for backward compatibility, the advanced channel `demand` is provided.
```

Comment:
Indeed, I discovered that when working on thermostats. The code was there, so this might never have been added, or have been lost somewhere. I removed the 'for backward compatibility', but suggest to keep the channel, in sync with the other numeric advanced channels.